VultisigApp/VultisigApp/Model/Cosmos/KeysignPayload.swift

```diff
@@ -29,8 +29,8 @@ enum BlockChainSpecific: Codable, Hashable {
         switch self {
         case .UTXO(let byteFee, _):
             return byteFee
-        case .Ethereum(let baseFee, let priorityFeeWei, _, _):
-            return baseFee + priorityFeeWei
+        case .Ethereum(let baseFee, let priorityFeeWei, _, let gasLimit):
```

Comment:
please separate it's into other property, changing gas can break logic for other callers